backend/tests/benchmark_urgency.py

```
@@ -18,22 +18,22 @@
     for _ in range(iterations):
-        # We only benchmark _calculate_urgency. We give it a base severity of 10.
-        priority_engine._calculate_urgency(sample_text, 10)
+        # We only benchmark calculate_urgency. We give it a base severity of 10.
+        priority_engine.calculate_urgency(sample_text, 10)
     end_time = time.perf_counter()
 
     total_time = end_time - start_time
     avg_time_ms = (total_time / iterations) * 1000
 
-    print(f"Benchmark: _calculate_urgency")
+    print(f"Benchmark: calculate_urgency")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)